### DISCUSSION TITLE
IMonacoEditorOptions to chose line endings

### DISCUSSION THREAD

**Discussion by brism17** [microsoft/monaco-editor#3694] [Q&A]
It would be nice if there was a monaco editor option to control the line endings. Currently the default value is for CRLF but I would like to be able to chose LF line endings for the editor . Is this possible currently ?

**Comment by rcjsuen** [comment]
@brism17 Maybe try [`pushEOL`](https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.ITextModel.html#pushEOL)?

**Comment by cirosantilli** [comment]
I'm unable to understand the behaviour of `pushEOL`/`setEOL`, either buggy or very weird: https://github.com/microsoft/monaco-editor/issues/3440
